### PR TITLE
Updating README

### LINKED ISSUE #1744: Update README based on the new updates of the repo

### PR DESCRIPTION
**Pull request by HristoP96**
Closes #1744

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,12 @@
 
 This project is using Ignite UI for Angular components to demonstrate different scenarios related to the usage of each component. The project is tightly related to [Ignite UI DocFX Site Builder](https://github.com/IgniteUI/igniteui-docfx). All samples are part of the official component topics from our [website](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid.html).
 
+The project is divided into two two separate applications:
+
+- A `default application`, which uses all the [igniteui-angular](https://travis-ci.org/IgniteUI/igniteui-angular) components to demonstrate sample applications
+
+- A `line of business` application, which demonstrates enhanced application scenarios, using igniteui-angular-charts, igniteui-angular-excel library, igniteui-angular-sparklines combined with `igniteui-angular` components.
```
Comment (hanastasov) [inline README.md]: Line of Business

Comment (hanastasov) [reply README.md]: IMO it is better to describe it like the igniteui-angular has been complemented with the charts and excel libraries, smth like: "A 'Line of Business' application complemented with UI and functionality from the Ignite UI for Angular Charts and Ignite UI for Angular Excel libraries"

File: README.md [hunk 2]
```diff
@@ -85,4 +148,4 @@ npm run build:lib
 
 In **igniteui-angular-samples**:
 
-- paste the aforementioned copied, built `igniteui-angular` folder in the `node_modules` directory
+- replace the copied `igniteui-angular` with its identical folder in the `node_modules` directory
```
Comment (hanastasov) [inline README.md]: identical or corresponding ?

File: README.md [hunk 3]
```diff
@@ -5,6 +5,12 @@
 
 This project is using Ignite UI for Angular components to demonstrate different scenarios related to the usage of each component. The project is tightly related to [Ignite UI DocFX Site Builder](https://github.com/IgniteUI/igniteui-docfx). All samples are part of the official component topics from our [website](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid.html).
 
+The project is divided into two two separate applications:
```
Comment (zdrawku) [inline README.md]:
```suggestion
The project is divided into two separate applications:

```

**Pull request merged**